### PR TITLE
Update build script

### PR DESCRIPTION
**Pull request by fisker**

### CODE REVIEW

File: src/needs-parens.js [hunk 1]
```diff
@@ -92,8 +91,6 @@ function needsParens(path) {
           }
 
           if (pp === np && key === "right") {
-            assert.strictEqual(parent.right, node);
```
Comment (fisker) [inline src/needs-parens.js:95]: This actually meanless, since `key === "right"` already ensure it's accessing `parent.right`.

**Comment by czosel** [comment]
@loilo could you take a look at this?

File: src/needs-parens.js (hunk 1 above)
Comment (loilo) [reply src/needs-parens.js:95]: I mean, I kind of fear that we could be running into a Chesterton's Fence problem here. But I don't see any case where that line would be needed either. 🤷‍♂️

**Review by loilo — COMMENTED** [review]
Looks all good to me. Standalone file builds and works fine, and also works in the playground. 👍

**Review by loilo — APPROVED** [review]

**Pull request merged**